docs/api-guide/fields.md

```diff
@@ -124,6 +124,8 @@ A boolean representation.
 
 When using HTML encoded form input be aware that omitting a value will always be treated as setting a field to `False`, even if it has a `default=True` option specified. This is because HTML checkbox inputs represent the unchecked state by omitting the value, so REST framework treats omission as if it is an empty checkbox input.
 
+Note that default `BooleanField` instances will be generated with a `required=False` option (since Django `models.BooleanField` is always `blank=True`). For JSON-only APIs you may wish explicitly declare BooleanFields in order to control this. ([An example configuring serialiser generation to use a `BooleanField` subclass with `required=True` may be found here][required-by-default-boolean-field].)
```

Comment:
* "If you want to change this behaviour you should explicitly declare the BooleanField on the serializer class. (For example, JSON-only APIs where you do not want to allow empty values by default against boolean fields.)"
* Let's drop the "an example configuring..."

Reply:
Will re-word now. You want to drop the link to the example entirely?